charts/kuberpult/values.yaml

```diff
@@ -253,6 +253,9 @@ auth:
     baseURL: ""
     # List of scopes to validate the token. Please add them as comma separated values.
     scopes: ""
+  iapAuth:
+    # enfore IAP authorization for API
```

Comment:
this requires a bit more explanation as to why it's there. I'm not sure if "enforce" is the best word to describe it.